tests/componenttests/server/tests/mediaPipeline/CreatePipelineTest.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2023 Sky UK
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "MediaPipelineTest.h"
+
+namespace firebolt::rialto::server::ct
+{
+/*
+ * Component Test: Create Media Pipeline Sequence
+ * Test Objective:
+ *  Test the media pipeline can be successfully created and destroyed
+ *
+ * Sequence Diagrams:
+ *  Create, Destroy - https://wiki.rdkcentral.com/display/ASP/Rialto+MSE+Misc+Sequence+Diagrams
```

Comment:
Incorrect link, should be https://wiki.rdkcentral.com/pages/viewpage.action?pageId=226375556

Its in many of the descriptions, and same on client side my bad :)

Reply:
done